base-theme/assets/webpack/webpack.prod.ts

```diff
@@ -12,7 +12,7 @@ const prodOverrides: Configuration = {
 
   output: {
     path:       path.join(__dirname, "../../dist/static_shared"),
-    publicPath: "/static_shared",
+    publicPath: "/static_shared/",
```

Comment:
I think publicPath issues were the cause of the error you mentioned in Slack.

This fixes it for the production build. Fixing it for the development build is going to take a bit more effort, and something I want to talk a bit with Carey about in the office tomorrow. 

I think we'll **need** to address these publicPath issues if we decide to go the dynamic import route. We probably *should* address them either way.